src/config.rs

```diff
@@ -443,105 +544,62 @@ impl Cfg {
             .transpose()?)
     }
 
-    pub(crate) fn find_override(
+    pub(crate) fn find_active_toolchain(
```

Comment:
I think this rename should probably in a commit by itself?

Reply:
@djc I'm not so sure about that... Although it could possibly make the PR easier to review, the intermediate state that you're looking for might have inconsistent semantics (that is, a "new" type/fn name with an "old" implementation which cannot match that name). IMHO it's like intentionally introducing bugs into the codebase 🤔